actors/actors.go

```diff
@@ -0,0 +1,82 @@
+package actors
+
+import (
+	cid "github.com/ipfs/go-cid"
+
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	"github.com/filecoin-project/specs-actors/actors/builtin/account"
+	"github.com/filecoin-project/specs-actors/actors/builtin/cron"
+	init_ "github.com/filecoin-project/specs-actors/actors/builtin/init"
+	"github.com/filecoin-project/specs-actors/actors/builtin/market"
+	"github.com/filecoin-project/specs-actors/actors/builtin/miner"
+	"github.com/filecoin-project/specs-actors/actors/builtin/multisig"
+	"github.com/filecoin-project/specs-actors/actors/builtin/paych"
+	"github.com/filecoin-project/specs-actors/actors/builtin/power"
+	"github.com/filecoin-project/specs-actors/actors/builtin/reward"
+	"github.com/filecoin-project/specs-actors/actors/builtin/system"
+	"github.com/filecoin-project/specs-actors/actors/builtin/verifreg"
+)
+
+type BuiltinActor struct {
+	exports []interface{}
```

Comment:
Rather than call Exports() for this list, reference a `Invokee` reference (which is an `x.Actor{}` object), which will be more flexible if we ever need to extend that.

And in fact BuiltinActor could then implement `Invokee`.